src/FlatbufferMessage.h

```diff
@@ -54,7 +54,7 @@ class FlatbufferMessage {
   ///
   /// \param BufferPtr Pointer to memory location containing flatbuffer.
   /// \param Size Size of flatbuffer in bytes.
-  FlatbufferMessage(char const *BufferPtr, size_t Size);
+  FlatbufferMessage(uint8_t const *BufferPtr, size_t Size);
```

Comment:
Can you please motivate this change? I.e. what do we gain and why is it better than what we had before.

Reply:
I added ~4 reinterpret casts and removed ~20. `uint8_t` is what flatbuffers uses internally.

Reply:
👍